README.md

```diff
@@ -68,15 +68,15 @@ def main():
     encoding = bitmovin.encodings.Encoding.create(encoding).resource
 
     video_codec_configuration_1080p = H264CodecConfiguration(name='example_video_codec_configuration_1080p',
-                                                             bitrate=2400000,
+                                                             bitrate=4_800_000,
```

Comment:
Please remove the underscores, as they are support in Python >= 3.6 only. Our minimum requirement is 3.3
https://www.python.org/dev/peps/pep-0515/